testing/features/buttons.feature

```diff
@@ -1,14 +0,0 @@
-Feature: Buttons components
-
-  The Button components gives the current site a variety of means of
-  providing means to submit data
-
-  NB: Due to these buttons being very visual and the behaviour changing
-  dependent on each browser, we will validate them with some rigorous VRT
-  tests later on, and not test any behaviour here
-
-  Background:
-    Given the Buttons component is on the page
-
-  Scenario: Validate initial state
-    Then all the buttons are present
```

Comment:
Can we leave this feature/scenario present, as it gives us some BDD description of the feature.

Reply:
This doesn't _do_ anything really right now, but sure happy to leave it in.

Reply:
Hmm scratch that, because the new docs format has one page per button variant, these tests don't make sense as is. We could either write three different "page" tests, or leave them out (which is my preference here)

Reply:
Ah right, because the button components are on a different page to what was previous right?

Reply:
Not just on a different page, but technically _three_ different pages (one for each variant). That's the way stories ideally work in storybook (we previously squished everything into a single story)